Sources/AST/AST.swift

```diff
@@ -203,6 +203,10 @@ public struct FunctionDeclaration: SourceEntity {
 
   public var mangledIdentifier: String? = nil
 
+  public var isVoid: Bool {
```

Comment:
I can't do this directly because the resultType isn't set for all function types hence it's optional. I've added this check in addition to a null check.